src-script/download-artifact.ts

```diff
@@ -20,7 +20,7 @@
 // This util script downloads ZAP artifact from Github.com
 // Usage: node ./download-artifact.js $branch [ $commit | latest ]
 
-import { Octokit, App } from 'octokit'
+import { Octokit } from 'octokit'
```

Comment:
@SaharKargar
Can you rebase your changes?
There are a few changes that is not related to data-* work.

Reply:
@jingteng25742 
Rebased and pushed.